commands-yml/commands/interactions/actions.yml

```diff
@@ -0,0 +1,165 @@
+    WebElement target = (MobileElement) driver.findElementsByAccessibilityId("SomeOtherAccessibilityID");
+
+    Actions actionBuilder = new Actions(driver);
+    Action dragAndDropAction = actionBuilder.clickAndHold(source)
+                                            .moveToElement(target, 1, 1)
+                                            .release(target)
+                                            .build();
+    dragAndDropAction.perform();
+  python:
+    |
+      element = driver.find_element_by_accessibility_id("elId")
+      actions = ActionChains(driver)
+      actions.move_to_element(el)
+      actions.click(hidden_submenu)
+      actions.perform()
+  javascript_wd:
+    |
+      // Not supported
+  javascript_wdio:
+    |
+      // Example: expressing a 1-second pinch-and-zoom
+      // with a 500ms wait after the fingers first touch:
+      browser.actions([{
+          "type": "pointer",
+          "id": "finger1",
+          "parameters": {"pointerType": "touch"},
+          "actions": [
+              {"type": "pointerMove", "duration": 0, "x": 100, "y": 100},
+              {"type": "pointerDown", "button": 0},
+              {"type": "pause", "duration": 500},
+              {"type": "pointerMove", "duration": 1000, "origin": "pointer", "x": -50, "y": 0},
+              {"type": "pointerUp", "button": 0}
+          ]
+      }, {
+          "type": "pointer",
+          "id": "finger2",
+          "parameters": {"pointerType": "touch"},
+          "actions": [
+              {"type": "pointerMove", "duration": 0, "x": 100, "y": 100},
+              {"type": "pointerDown", "button": 0},
+              {"type": "pause", "duration": 500},
+              {"type": "pointerMove", "duration": 1000, "origin": "pointer", "x": 50, "y": 0},
+              {"type": "pointerUp", "button": 0}
+          ]
+      }]);
+
+      // release an action
+      browser.actions();
+  ruby:
+    |
+      # Send keys to an element
+      action_builder = driver.action
+      keyboard = action_builder.key_input
+      el = driver.find_element(id: "some_id")
+      driver.action.click(el).pause(keyboard).pause(keyboard).pause(keyboard).send_keys('keys').perform
+
+  php:
+    |
+      // TODO
+  csharp:
+    |
+      // TODO
+
+client_docs:
+  java: "https://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/interactions/Actions.html"
+  python: "https://selenium-python.readthedocs.io/api.html#module-selenium.webdriver.common.action_chains"
+  javascript_wdio: "http://webdriver.io/api/protocol/actions.html"
+  #javascript_wd: "https://github.com/admc/wd/blob/master/lib/commands.js#L44"
+  ruby: "https://www.rubydoc.info/gems/selenium-webdriver/Selenium/WebDriver/W3CActionBuilder"
+  php: "https://github.com/jupeter/selenium-php-webdriver/blob/master/Webdriver/Common/ActionChains.php"
+  csharp: "https://github.com/appium/appium-dotnet-driver/" # TODO
+
+# Driver support by platform
+driver_support:
+  ios:
+    xcuitest: false
```

Comment:
xcuitest only supports pointers of type touch and nothing else

Reply:
@mykola-mokhnach I updated it to `true`... when we do a code sample we can make a note about it not support key events on iOS.